README.md

```diff
@@ -17,12 +17,12 @@ Before using any of our Docker images, you'll need to create a .env file. The re
 
 `docker-compose up` will build and install our images, and launch them in containers.  These images are:
 
-* db: The MySQL database storing NYC city data both in semi-raw form and joined into a model for the Rails website. The database can be examined with `docker exec deltanycheatseek_db_1 mysql -u <user from .env> -p <password from .env>`. Local clients can connect at 127.0.0.1:3306 ([localhost will not work](http://stackoverflow.com/a/32361238/103315)).
+* db: The MySQL database storing NYC city data both in semi-raw form and joined into a model for the Rails website. The database can be examined with `docker exec deltanycheatseek_db_1 mysql -u <user from .env> -p <password from .env>`.
```

Comment:
Note: I removed the mysql port so it wouldn't be exposed when running in production. It does remove a potential way to access the DB locally, but I don't think too many people were connecting that way.